lib/src/components/files/fileCard/stories/FileCardStory.js

```diff
@@ -59,15 +60,52 @@ stories.add('FileCard', () => {
   );
 
   return (
-    <StoryItem
-      title="FileCard"
-      description="A card component which utilises the thumb, meta and control modules to display a file."
-    >
-      <ul className="list-none grid p-0 m-0 tw grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4">
-        <li>
-          <FileCard preview={preview} meta={meta} insetMeta={insetMeta} />
-        </li>
-      </ul>
-    </StoryItem>
+    <>
+      <StoryItem
+        title="FileCard"
+        description="A card component which utilises the thumb, meta and control modules to display a file."
+      >
+        <ul className="list-none grid p-0 m-0 tw grid-cols-2 sm:grid-cols-3 md:grid-cols-4 gap-4">
+          <li>
+            <FileCard
+              preview={<Preview>{controls}</Preview>}
+              meta={meta}
+              insetMeta={insetMeta}
+            />
+          </li>
+          <li>
+            <FileCard
+              preview={<Preview>{controls}</Preview>}
+              meta={meta}
+              insetMeta={insetMeta}
+            />
+          </li>
+          <li>
+            <FileCard
+              preview={<Preview>{controls}</Preview>}
+              meta={meta}
+              insetMeta={insetMeta}
+            />
+          </li>
+          <li>
+            <FileCard
+              preview={<Preview>{controls}</Preview>}
+              meta={meta}
+              insetMeta={insetMeta}
+            />
+          </li>
+        </ul>
+      </StoryItem>
+
+      <StoryItem
+        title="FileCard (thumbnail usage)"
+        description="A card component which utilises the thumb, meta and control modules to display a file."
+      >
+        <FileCard
+          preview={<Preview loader={<Loader size="sm" />} />}
+          className="w-10"
```

Comment:
💚 Do you feel we should have a `thumbnail` prop or something similar that sets this width, rather than doing it on the fly?

Reply:
We could do, I'll take a look 👍

Reply:
@altaudio I don't think having a prop for a "thumbnail" size isn't ideal for `FileCard`. 

We'll end up setting props to parents just for children as `Loader` and `PreviewImage` have a number of props individually, which leads us to smell patterns in the future e.g. `props.loaderHeadingText`, `props.loaderProgressText`, `props.loaderSize` 🤮 

It also doesn't help that the size of the image preview is set outside of the component so setting a thumbnail prop will mean we'll have some implementation that is internally managed and then some that are externally managed. Making is harder to digest.